src/mod.ts

```diff
@@ -139,6 +143,12 @@ class PathToTarkov implements IPreSptLoadMod, IPostSptLoadMod {
 
   public postSptLoad(container: DependencyContainer): void {
     this.container = container;
+    this.instanceManager.postDBLoad(container);
```

Comment:
Is this line should be moved into a real `postDBLoad`  ? because we are in the postSptLoad here.